### DISCUSSION TITLE
PLease how to restream to icecast server fo radio

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6255] [FAQs] [answered]
PLease how to restream to icecast server fo radio
thanks

**Comment by Mohit-3196** [comment, marked as answer]
Hi @brixxnatt,
How are you?

Thank you for reaching out.
I do not have experience with the Icecast server but I can check this over the weekend and update you.
I hope that is fine with you.

Thank you.

**Comment by brixxnatt** [comment]
thank you

**Reply by Mohit-3196** [reply]
Hi @brixxnatt,
I discussed this and saw that the Icecast server supports audio streaming and therefore, we need to transmux the output.
FFmpeg can do this with the following command
`ffmpeg -re -i rtmp://192.168.22.12/live/live1 -vn \
       -codec:a libmp3lame -b:a 64k -f mp3 \
       -content_type audio/mpeg \
       icecast://source:pass@example.com:8000/mountpoint.mp3`

If you would prefer to have a solution on the Ant Media Server side then we can discuss the scope of the project and accordingly, assign a priority for this to move ahead.

Thank you.
